mono/sgen/sgen-gc.h

```diff
@@ -15,6 +15,9 @@
 /* pthread impl */
 #include "config.h"
 
+#include <stddef.h>
+void sgen_qsort (void *array, size_t count, size_t element_size, int (*compare) (const void*, const void*));
```

Comment:
Because a test uses it, w/o the define. It is not prototyped -- warning in C, error in C++.
Would rather not have the larger change of adding the define to the test.

Reply:
Also, declarations don't have a huge semantic or detectable meaning -- not clear the `#ifdef` is even needed. Ultimately the link succeeds or fails shortly thereafter.

Reply:
That was my mistake, sorry, of testing in the not fully C++ tree. In the fully C++ tree,  the #define works so let's restore the sgen .h and define in the test.